testsuite/src/it/java/io/quarkus/ts/startstop/utils/WhitelistLogLines.java

```diff
@@ -77,6 +82,25 @@
             // (no explicit configuration, none or multiple JDBC driver extensions)
             // Result of DevServices support https://github.com/quarkusio/quarkus/pull/14960
             Pattern.compile(".*Unable to determine a database type for default datasource.*"),
+    }),
+    WINDOWS_COMMON(new Pattern[] {
+            // When running on Windows, some warning SLF4J traces are randomly printed
+            Pattern.compile(".*SLF4J: .*")
```

Comment:
I didn't report this issue as I can't reproduce it locally and it's only printed randomly and as warnings, so it's not a big issue.

Reply:
This is weird, just on Windows? We had issues with leaking of this warning on Linux some time ago too.

We have explicit check for sl4j - https://github.com/quarkus-qe/quarkus-startstop/pull/45/files

Reply:
I reverted these changes.